tests/core/cgo/BUILD.bazel

```diff
@@ -52,16 +52,19 @@ go_library(
     copts = [
         "-DRULES_GO_C",
         "-I$(GENDIR)/tests/core/cgo/generated_copts",
+        "-I$(GENDIR)/external/io_bazel_rules_go/tests/core/cgo/generated_copts",
         "-DDOLLAR_SIGN_C=$$",  # the dollar sign should be escaped
     ],
     cppopts = [
         "-DRULES_GO_CPP",
         "-I$(GENDIR)/tests/core/cgo/generated_cppopts",
+        "-I$(GENDIR)/external/io_bazel_rules_go/tests/core/cgo/generated_cppopts",
```

Comment:
can you use https://docs.bazel.build/versions/main/skylark/lib/native.html#repository_name and https://docs.bazel.build/versions/main/skylark/lib/native.html#package_name perhaps ?

Reply:
Thanks. I did not know about these. PTAL.